app/src/main/res/values-da/strings.xml

```diff
@@ -1625,4 +1625,5 @@ the Select` button. When clicked it closes all the previously selected tabs -->
     <string name="settings_language_galician">Galicisk</string>
     <string name="permissions_dialog_remember_choice">Husk denne beslutning</string>
     <string name="settings_language_portuguese_br">Portugisisk (Brasilien)</string>
-</resources>
\ No newline at end of file
+    <string name="settings_language_portuguese">Portugisisk</string>
```

Comment:
Mind moving this to the place where the other languages are? It's fine to move the portuguese_br as well.

Same comment for all the languages.

Reply:
Note that the Weblate tool we are using adds new translations to the end of the xml files. It also strips down the comments and do all kinds of cleaning and re-formatting.

I'm all for keeping strings organized in this case, but I think in general it will not be practical to do manual editing of these XML files after Weblate has touched them.

Reply:
My point is that as we add new strings, Weblate will keep appending them at the end of each and one of these XML files. So the expectation of having strings grouped there is a bit moot, and a waste of effort at best.

Reply:
Ah OK, fair enough. Let's merge it then